flake.nix

```diff
@@ -0,0 +1,25 @@
+{
+  description = "A very basic flake";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-21.11";
+  };
+
+  outputs = { self, nixpkgs }: 
+  let
+    pkgs = import nixpkgs { system = "x86_64-linux"; };
+  in
+
+  {
```

Comment:
forAllSystems (or use flake-utils). No reason to have this flake be x86_64-linux specific.